src/riak_ensemble_client.erl

```
@@ -105,3 +125,12 @@ translate(Result) ->
             %%       Perhaps build this logic into peer to return {error, notfound}?
             Result
     end.
+
+-spec maybe(node(), fun()) -> client_reply().
+maybe(Node, Fun) ->
+    case catch rpc:call(Node, riak_ensemble_manager, enabled, []) of
```

You're correct. At the moment, however, `riak_client` actually crashes when used on nodes that aren't part of the consensus cluster -- something on the list to fix, but whatever. Folks using `riak_client` outside of Riak is something that's been deprecated forever, but we've tried not to break it. I don't particularly care for breaking it for "new in 2.0" consistent operations.

Nonetheless, the only safe way for `riak_client` to be used for consistent K/V operations currently ends up making it so `Node` will always be `node()`. So, how about special case the maybe check?

``` erlang
check_enabled(Node) when Node =:= node() ->
    riak_ensemble_manager:enabled();
check_enabled(_) ->
    true.
```

So, we'll still fail fast in the way Riak uses `riak_ensemble`, but be safe in general for other uses of this library.